soc/arm/nxp_imx/rt/soc.c

```diff
@@ -213,6 +213,13 @@ static ALWAYS_INLINE void clock_init(void)
 	CLOCK_SetMux(kCLOCK_CanMux, 2); /* Set Can clock source. */
 #endif
 
+#if defined(CONFIG_FLASH_MCUX_FLEXSPI) && !DT_PROP(DT_NODELABEL(flexspi), xip)
+	CLOCK_DisableClock(kCLOCK_FlexSpi);
+	CLOCK_InitUsb1Pfd(kCLOCK_Pfd0, 24);
+	CLOCK_SetMux(kCLOCK_FlexspiMux, 3);
+	CLOCK_SetDiv(kCLOCK_FlexspiDiv, 2);
```

Comment:
I have tested the FlexSPI driver on board [VisionSOM-RT](https://wiki.somlabs.com/index.php/VisionSOM-RT) with i.MX RT1052. When FlexSPI root clock is equal 120 MHz, amplitude of signal FlexSPI_CLK is too small. I attach two oscilloscope screenshot, with `FlexSpiDiv==2` and `FlexSpiDiv==3` (120 MHz and 90 MHz). It is depended on board capacity, so i suggest to use DTS for setup value of this divider.
![Div2](https://user-images.githubusercontent.com/73971382/104431189-a8c7fa00-5598-11eb-87a8-c849eca7bff0.png)![Div3](https://user-images.githubusercontent.com/73971382/104431191-a9609080-5598-11eb-847f-3622395a6f60.png)